lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -237,6 +238,28 @@ def _get_congestion_control_config(service_mconfig):
     return True
 
 
+def _get_converged_core_config(service_mconfig):
```

Comment:
Add typing annotations for arg type and return type

Reply:
Sure, I will do that.

Reply:
Generally speaking: when you update or add new Python functions, include type annotations